event_sink_clickhouse/sinks/base_sink.py

```diff
@@ -0,0 +1,26 @@
+"""
+Base classes for event sinks
+"""
+
+from django.conf import settings
+
+
+class BaseSink:
+    """
+    Base class for ClickHouse event sink, allows overwriting of default settings
+    """
+    def __init__(self, connection_overrides, log):
+        self.log = log
```

Comment:
why do we need the log to be part of the class?

Reply:
The next PR will be to add a management command that will call into here, so I'm using the pattern established in Coursegraph that passes in the log so that it can go to the celery log or normal IDA log based on how it's being run.